challenges/2021_cloud_base_height/03_pytorch_train.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
<p>This is not something I've thought about before, but should we saving these normalisations with the run in ML Flow do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes it would be desirable to save these in MLFlow runs, I do save the method name as a hparam, overall this is a difficulty though. to log this file as an artifact every time doesn't seem correct (its the same file everytime), and there doesn't seem to be a method of logging to the experiment itself without having it be a run which I think is most desirable.